internal/services/group_service_test.go

```
@@ -696,6 +696,66 @@ func TestCreateGroupRejectsGatewayProxyWithUpstreamProxy(t *testing.T) {
+	require.NoError(t, err)
+	var cachedUpstreams []struct {
+		ProxyURL     *string `json:"proxy_url,omitempty"`
+		GatewayProxy string  `json:"gateway_proxy,omitempty"`
+	}
+	require.NoError(t, json.Unmarshal(cached.Upstreams, &cachedUpstreams))
+	require.Len(t, cachedUpstreams, 1)
+	require.NotNil(t, cachedUpstreams[0].ProxyURL)
+	assert.Equal(t, manualProxyURL, *cachedUpstreams[0].ProxyURL)
+	assert.Empty(t, cachedUpstreams[0].GatewayProxy)
+
+	updated, err = svc.UpdateGroup(context.Background(), group.ID, GroupUpdateParams{
```

golangci-lint (2.12.2)

[error] SA4006: this value of updated is never used

(staticcheck)